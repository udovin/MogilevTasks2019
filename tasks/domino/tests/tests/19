515
-1571758 4863
-17126550 29918
-33479083 94798
-27300353 712
-17503805 62182
-15050649 21596
-7559811 21698
-34830617 4586
-3715524 35400
-18764917 112914
-16957355 104491
-26796110 66
-5093874 6585
-20375510 60620499
-28001439 9073
-1415104 1389
-16396521 14870
-34248080 28203
-9887232 31861139
-13712817 60998
-1817615 12574
-27089544 109962
-30259200 36928
-1891109 103692
-22698773 76602
-28746926 1422
-10471148 789
-2894834 7444
-24791524 57978
-17859796 66640
-2910538 16161
-29596038 31745
-5364437 727
-23287407 26467
-19870069 44487
-18377337 14736
-32469198 32446
-3399177 87481
-16220791 3428
-27442708 70954
-3865760 5107
-17475381 4440
-32636411 61169156
-30584712 8612
-21404904 23810
-34087853 16185
-3168159 55310
-34953022 709
-7972646 33877
-31570080 24832
-18260586 32525
-20292895 4603
-2729271 5947
-27273071 7370
-20184780 59245
-13603120 2350
-19265650 34746
-9159379 3027
-29268831 78686
-19649600 13268
-15395170 18967
-8894594 73888
-13823062 45460
-26423547 45437
-32760964 2775
-12586701 7107
-32341529 92323
-14937841 23032
-14699295 70684
-24382184 1077
-25944885 89848
-10131760 6844
-10479903 4758
-13285146 50772
-28012079 94654900
-16470926 7006
-29795385 29856
-26544216 91770
-30794000 112779
-4147090 43234
-15242914 17833
-19584769 27413
-33245586 11761
-22299314 1099
-34084276 2935
-1017053 15492
-26259977 46129
-25903743 3879
-30600378 20558
-1308174 79403
-34758975 1671
-34953812 11070
-23099368 28073
-7255366 12408
-5119629 9197
-10848402 1243
-21343848 856
-33027527 26391
-25638066 7971
-11458003 38806
-14242891 2748
-23026253 67197
-4810420 6254
-22184631 114438
-5215930 15426
-31889776 23810
-8860809 9529
-15821574 42071
-19129422 34626
-5205687 7032
-12299041 27150
-1417563 16254
-13051825 87056
-26346384 16955
-23440689 188
-11171054 71110
-16575521 91588
-30016940 57791
-29348537 14841
-1860462 7816
-10275662 31493
-25761296 31431
-28868620 8509
-31289222 88728397
-5238673 34425
-7964340 7517
-34908792 2799
-22823545 950638
-5987704 1561
-15629828 26243
-20948845 5253
-8447391 36702
-22977839 40984
-25231416 19759905
-5793329 41013
-16162386 14651
-7250439 2346
-20772047 8817
-29659672 1998
-9373440 4430
-1586419 28811
-31376863 369
-20300939 37255
-4665729 5470
-15707827 93967
-26521208 10261
-14493188 33665
-26670521 15455
-8654499 5389
-7016738 22458
-19004168 124054
-34850088 38169
-32010128 52387
-23573992 53958
-19701307 11666
-7607263 90046
-6858505 19985
-28907676 87320
-28589295 11256
-7808515 95161448
-27305398 51656
-1205168 8055
-2562594 33165
-6316829 17219
-29054899 72795
-10710770 59339
-30390162 6691
-23762457 3220
-9472414 39802
-12845086 22592
-28117953 98839
-21656750 22515
-21450246 1517
-26952192 52696
-13476409 46724
-21032932 105219
-32561992 22153
-13281894 2676
-16829927 57370
-16919249 24510
-27799017 33059
-35064587 1219
-23346573 2897
-32251899 44373
-12583544 1887
-16308172 4412
-34119875 64817
-25102055 19189
-19844602 3905
-24673236 46639
-1746318 83270701
-26689036 12722
-20522497 46919
-15986987 1393
-7884130 61013
-12042047 46498842
-16108868 4848
-15547413 6506
-5101075 11627
-15478598 27446
-28795765 50781
-17990435 48150
-25499316 8477
-33933552 37321
-9052203 29730
-12535388 39500
-14126384 94152
-6825875 12391
-9976468 9893
-20972246 3015
-17233933 32544230
-34059924 3491
-17373790 53
-28620551 21702
-15082438 83542
-8564088 8678
-9782779 3447
-30159181 6714
-28659892 51960
-8304773 3110
-25475396 15137
-25221794 6822
-30918321 7856766
-10591889 45338
-23547833 22759
-7172322 37137
-16125359 175
-25157489 16901
-4589175 571
-7062416 86253
-3350018 20568
-9315397 38917
-21522076 129
-11735154 56234
-5749284 13245
-25124167 14449
-5581822 72304597
-32317950 5613
-27212291 26552
-12675722 7588
-6840717 11771
-2604826 53290
-21146749 105741
-15621019 2346
-18041725 26444054
-22063371 12039
-13453253 13902
-22436844 97354778
-11852729 13523
-34430128 77751
-31463543 41634
-6047577 57392
-16788448 11786
-5401785 12631
-9208655 65260
-20057876 5780
-25072847 1137
-31721047 36516
-17780401 2284
-15989421 81431
-12437123 58150
-10042170 21566
-30646012 59180
-34302034 10875567
-4678392 49433
-34186033 61995
-22792971 6777
-1032636 60490
-29004399 34476
-14268882 89295
-23441929 89909
-33800956 29328899
-24490288 36440
-26057381 2864
-2836372 23980
-29145612 24973
-29909377 39934
-6554710 10009
-26796398 19124
-5708599 27443
-8089728 68948
-2464394 14230
-17664695 28846833
-1137124 1600
-3227274 83233
-1235838 97689524
-31612231 4890
-28470898 7484
-33095752 23284
-20850455 12785
-12685200 11103
-22895190 34627
-30364625 799
-10418658 16019
-4902825 33327
-34761739 64879
-23905377 43091
-1675233 7340
-5472300 44308
-17607129 24522
-1532023 2179
-23352118 22278
-30195332 47298
-11057891 67396
-2295098 907
-13960511 95994
-19472406 11348
-10339052 56400
-22098290 70801
-23835079 23128533
-15424710 3868
-27658288 28203
-6918339 15865
-18888680 37703
-13608746 38084809
-15201914 34593
-18164447 34141
-31159816 1454
-12386094 36876
-11275153 28487
-33602598 9284
-20865677 26520
-17742433 17428
-2060782 119777
-3061349 22773
-5888706 86807
-10833239 14573
-7050706 2521
-24893619 51396874
-25657709 8239
-31819090 7068
-21263751 6676
-21457065 64559
-10827026 5703
-27689714 61876
-18669622 44688
-27402860 18905
-33157759 40127
-24536298 73894
-24990441 62847
-6470699 64781
-20669208 31727
-19716034 1696
-22811548 2076
-6605066 76618507
-6713200 70428
-4514576 52402
-17377497 70741
-23169231 39013
-27840830 34093
-28078821 28968
-13586315 14036
-17067164 54902
-19372698 45798
-31642783 2998
-7408708 64476
-22542683 19263
-18951352 20772
-11674828 45718
-25891213 4256
-24169824 19379
-17349221 6480
-9885905 398
-28241647 91095
-11925522 79651
-16741443 1413
-4005170 15020
-10157536 64641
-9598052 80894
-7321247 49344
-24437970 31847
-21732802 10018
-24050957 101421
-22026300 11071
-15703173 1791
-24224615 37659
-20978858 24689
-12964693 63192
-11574710 24842795
-16412492 24749
-31209372 4858
-8574325 21939
-31030959 4722
-29220264 5103
-28583334 817
-22624983 29151
-25537968 49787
-23636918 11450
-2316576 36746
-1438753 46690
-34670504 15636
-17169585 52182
-27552764 27113
-22939188 1829
-19205355 24646
-16126578 576
-32765381 9080
-27910069 52581
-32130400 61776
-21953786 18750
-13174138 73542
-33130752 19362
-29960974 31799
-26708508 300
-14810623 55960
-32067434 5076
-12601249 39897480
-26870051 46013117
-15273383 66600657
-16080249 20153
-33376703 53783631
-4395388 56919
-6237070 9112
-26071774 82354
-4065874 51013
-6347947 13316
-30471426 18629
-17663600 284
-26852881 7650
-13362576 64287
-14079207 9191
-21295609 7113
-9701498 71931
-21551486 95810
-3886618 78474
-2005222 27447
-31221746 32149
-3871330 631
-2960482 51892
-15365994 2893
-33623465 24293
-21835279 2828
-25366256 28201
-25321909 27648
-1704729 33006
-14602247 75296362
-17975655 4347
-18525590 27731
-2185904 90360
-4234974 12982
-16146238 2650
-12124715 83579
-4992949 58050
-24625597 33421
-18585927 35452
-4790609 2950
-34633499 20234
-24309409 69173
-34556181 30268
-29490526 25656
-16676866 20027
-15171120 25816
-8384705 26721
-18400853 13448
-25824417 98429072
-7702803 66508
-23219897 37614
-12283622 9003
-33867009 60905
-24275940 23163
-6112780 70423
-10977602 60298
-23129105 31976
-18426442 24070
-15917113 9264593
-34970658 31544266
-8308031 11117
-33324323 2777
-13896299 16500
-33716753 8661
-32906795 12246
-3758896 95394
-1485611 4001
-33725992 53708
-14374432 78803
-32796662 1118
-8199878 72997
-30726568 62525
-4820442 55634
-29742429 18412
-33663402 7329
-12212649 19472
-3523202 44090
-6009122 37646
-28351149 16234
-3639971 73451
-29380076 35746
-30578808 3623
-11370707 52717
-8987825 51500
-30094259 30109
-26179730 5666
-31381440 41401
-31041750 25180
-20067685 109712
-13876950 14087
-1664926 8916
-2354059 40014
-27044255 29996
-32511751 28631
-20462995 52008
-22344376 85159
-34836949 4150
-23950635 73857
-19667535 6439
-8748574 111316
-20573780 55404
-7530134 27369
-8300440 2830
-9364616 5827
-12769379 32973
-4319255 26133
-19994153 17783
-8597464 15092
-25318513 2312
