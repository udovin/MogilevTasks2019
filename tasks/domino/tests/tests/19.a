1 1 1 1 2 1 1 1 1 2 2 1 1 285 1 1 1 1 133 1 1 2 1 3 2 1 1 1 1 1 2 1 1 1 2 1 1 2 1 2 1 1 474 2 1 2 1 1 2 1 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 2 3 1 1 1 3 1 1 2 407 1 1 2 2 1 1 1 1 1 1 1 1 1 3 6 1 2 1 2 1 1 1 1 1 1 1 3 1 3 2 1 1 1 1 1 2 2 1 1 1 1 1 2 1 1 1 1 1 455 3 1 1 322 1 2 1 1 2 362 1 1 1 1 1 1 2 1 2 1 4 1 1 1 1 1 3 3 1 2 1 3 2 2 2 103 3 1 1 1 2 1 1 1 1 1 2 1 1 1 2 4 1 1 2 1 1 1 1 1 1 1 4 1 1 1 17 1 1 1 1 159 1 1 2 1 2 2 1 1 1 1 2 1 1 1 240 1 1 1 2 1 1 1 2 1 1 1 449 1 1 1 1 1 1 3 1 1 1 1 1 1 71 1 1 1 1 2 1 1 254 1 1 316 1 1 1 3 1 1 2 1 1 1 1 218 2 1 4 501 2 1 1 2 1 2 4 492 1 1 1 1 1 1 2 1 1 1 248 1 2 5 1 1 1 1 2 1 1 1 1 2 1 1 1 1 1 2 2 1 1 3 1 1 2 340 1 1 1 1 184 2 1 1 1 1 1 2 1 2 1 1 2 1 355 1 1 1 3 1 2 1 1 2 2 1 1 1 1 1 85 1 1 3 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 1 3 1 1 1 2 1 154 2 1 2 1 1 1 1 2 1 2 3 1 2 1 1 1 1 2 1 1 1 1 1 1 1 1 1 169 390 205 1 485 1 1 2 1 1 1 1 1 1 1 1 1 5 3 1 2 1 3 1 2 1 1 2 2 196 1 1 1 1 1 160 1 1 1 1 1 2 1 1 1 1 1 1 372 1 1 1 1 1 4 1 2 1 216 514 3 1 1 1 1 2 1 2 1 1 1 1 3 1 1 1 1 2 1 1 2 1 1 1 1 1 2 2 3 1 1 3 1 1 1 2 1 2 1 2 2 1 1 1 1 1 1 1 1 
