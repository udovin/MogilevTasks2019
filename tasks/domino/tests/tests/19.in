477
-3864269 11253
12605123 110391
14758132 71045
9021234 1113
-10079182 61711
14446053 20901473
11207652 60799
15327044 56568
-7643902 44955
8897753 82790
4466333 61487
17578627 20986
-3800765 23445
19828792 9778
16842291 118859
7872761 73018603
-4121430 58312
15441655 19344295
18414455 33632
-11353971 5851
17536749 1965
-11173054 13790
-9526906 28755
-8254962 16441
-5501506 42598
-4588390 54443
1608456 38577
-11798153 53034
16660415 31155
-8283151 50880
15780619 20744
9755310 8250
14058193 10831
3603505 72862
-344183 52580
9665525 392
20011072 11894
15017300 824
15950347 85276
16274323 41126
7748410 24692
8187592 620
-977473 18869
4527038 25764
-850833 12811
-8237801 5102
16331532 923
10754334 79485
-9090297 68046
8957554 44008
8208762 10702
739361 82121
2809619 96337
11580017 56436
-13465158 21882
-11904832 18822
-10275095 15675
-10579072 16176
5252244 92431
-4791741 116
13075369 69892
9157476 24575
16971121 18191
18432369 2358
8754225 14633
4154186 63032
-12864351 39540
9029530 5571
6489521 61034
14987028 69915
12234044 33713
-6074050 5878
9013984 1180
-7321350 65
-13203874 86838
-4421215 30382
-1499974 75384
7609820 27540
19304922 25319
-11984572 42844
14317536 36716
6052213 34989
801122 2208
11962007 26093
10888707 31670
-7796164 50229
-11293381 20891
-4672299 71654
19216939 17550
4851635 55656
-9597143 9621
6677668 46308
13172522 2615
-2060603 39307
-13029392 26317
10040419 27255
-11028803 4257
-12564712 41487
5560179 10090
-10173813 24886
-13674349 49774
3621627 11326
17825366 70392
14507430 14483
-3295375 72104
-8572373 7479
-4341013 2460
-13084600 49411
-13305513 61872
3725530 58092644
-7691609 3018
-3415093 5459
-4519258 37413
-8942579 4280
-697816 62768
4047672 107712
1745925 51341
-11930255 13642
8078714 3563
-6362938 64143007
1469567 28770
-2916224 108072
-9214856 13172
19136059 76423
15464721 1043
-8586350 522
-10364832 44775
-8950705 16371
-1630272 66276
13637138 8598
15198885 35851
18140913 1528
5988187 113582
18318044 4326
19515277 82963
-11668419 39084
7075458 38562
-9333935 3985
11929118 17637
19415617 101175
15982092 4233
-9996888 41613
-10418081 31713
7029618 67630
-12336315 6841
14533077 16881
272951 44374415
-6854198 16131
6400941 9424
-7702498 50771
9851641 24878
-10680417 13894
15834277 30892
-1040878 46419
11479940 59879
18291685 24771
2051927 32315
10352511 23394
7237399 73107
5141474 32278
12127595 51123
9355219 3010
5632887 82148204
16601871 31790
15600462 48993
9339489 17871
16427583 39681
-11062385 23176
14880506 56371
2195333 8948
4648774 21059
1144708 25037
16327613 37306
18702944 1790
5786811 28411
2102058 26651
-12101512 4694
-840816 334
12378247 8571
339619 14060
-6691250 6805
-1935250 45949758
8774287 13041
8494156 15243
-10507150 4037
-5278528 95293
4534705 6043
7417795 11904
-10203506 7541
16081606 35268
15712578 89525
12636797 23456
7956121 40368
19173897 13096
16374683 5638
15107911 61070
-5499767 461
17874211 20860
-4051442 12312
11908733 17472
6707684 14858
10673081 35282
-481565 35288
15720603 152
13749594 77475
-9249711 26699
6572019 33950
10096722 39478
-13427285 18352
13487157 108129
1081640 47930
1620565 3281
-7505663 5148
-8838680 866
7370387 1413
-2175116 52645
-5387757 23270
-7388839 91638
-4932280 47853
2579848 58078
-109232 7824
16717441 20569
-9413304 2799
14533928 351
6073907 13344
-6169606 4016
9742222 1270
-11491836 41060
7534335 2620
7365657 60100
12751174 112006
3029247 44728
2970399 16515
4345649 25611
-4244602 13632
-9767550 7914
-2602285 35361
19260204 30015
4157075 1644
18051954 42859
-12850073 457
12974931 1863
19023030 35613
-7323722 29282
-7900515 48100
11340605 59354
-7171167 17478
18645956 77400
-5697859 15158
-5822891 26469
17205022 23956
906938 85361
13600843 69185
7106088 3164
2330097 42410
-8872006 15162
14205903 1294
-11315847 29087
6818976 59337
-9616039 1362
11519554 19423
-12675997 23105
8593755 34693
17518678 95297
20899 91521
8057919 78695306
2711718 22096
1214500 11565
7240121 1682
-12399620 54045
11699336 101283
-4390061 23666
-415192 50045
5745601 69571
12857988 17407
3796737 52088
2489733 52633
-6811749 18738
-12916372 68566
-9842199 4440
-2785479 64601
-1205964 124366
-9643414 7534
-6231421 9156
-12641903 26755
9072066 10908
-3205802 57630
16952815 26961
1662954 16389
-522864 6378
-7259811 19304
-8774113 12331
-1756464 100040
-6014677 21459
-579353 48378
-8490684 35848
5039163 56206040
6194721 22541
-13547714 117292
2668688 82899
651675 57122
12300470 10403
-221877 62727
18533855 16945
18202900 53958
12934702 70481
14330153 1107
11825467 65567
-8622325 91529
3205419 29755
8309990 2197
-12632060 4371
18130222 61882
-3024706 4341
10311540 98325
13296286 83171
-5187590 1335
9245665 10704
17985826 17399
-12280356 39022
415471 67029
7162999 54617
2273324 63265
1335944 61573
5350225 81436
-3509439 29188
10895390 2749
12250722 14630
-6990128 72486
-12491539 8720
-2233723 67756
-11583376 7694
2019275 74274
9423597 28631
14043929 92663
-9877327 113288
-10503912 3124
-6518621 50023
10173846 79989538
12027683 12227
19716898 29624
13877403 123267
-11609605 33247
-12849668 212
9969851 17499
1809851 31956
-8027043 45634
14818695 20234
14164468 54888
-10743140 5311
-12831201 17879
18869414 52340
4247348 80661
-7124198 44019
16487694 41874
-2723640 57955
16158180 26574
3517063 17568
4944973 83128408
-11127636 43603
14044996 928
1002725 62614
-1338120 58224
8737147 58339961
-6971967 2572
10801673 21375
13159369 319
3275661 64360
19880922 33776
10814533 2542
-3933147 43813
-6444842 5324
-1401528 65949183
10461306 42910
-9379934 2845
19153109 9235
-5417135 51550
19154035 538
8124726 20733
-2468573 65993
15965391 8653
1923394 92997
8605060 4312
9949908 72623
-11382260 54753
6171595 13229
-1144922 49161
6532027 13252
17412620 7215
-6765023 10250
3918066 63566
17541246 1670
-10899965 7933
9665095 5381
-310718 20175
10464977 1864
-1864102 4107
11034614 118976
9485369 10485
-794816 12580
-10737176 2536
4579762 7977
-10679889 67
-9655695 35550
153849 9723
-3088608 7839
12474433 60822
10590656 15452047
4780149 28377
18954853 57189
-10768129 104027
17142932 82388
9111296 5187
-2102715 32657
13347761 32712
8149486 11315
17027984 45853
-8359028 50707
-5587615 109408
17310049 1531656
4751718 52362
2856411 30382
18805677 6400
8288761 74967
6009527 21071
13940205 518
-12706423 82957
-5047373 34398
-11787464 2624
-12124988 79617
9605714 44522055
16618788 14734
-3033411 29933
3155294 74591
-8756493 13212
-3590558 56738995
195660 10531
17607349 20708
7957693 1551
-12508596 7325
5869575 36017
6391167 29673
19622383 58103
-8152548 32283
-7535501 3098
575865 61595
496800 23473
3434020 21676
-4824458 71116
-12854770 8953
-12246105 24068
5154139 2573
-6148328 17697
17693891 14559
8442724 85784
11139823 87283
6946992 91266
-8428909 44320
-6243386 112714
13158159 48799609
8605539 6
-3991683 30736
-9436592 87423
5481992 7663
-3695280 90496
12411678 31036
-3876475 45629
6315648 51011
-2334856 98498746
3354039 65271
2427012 10911
-6586887 9647
-5886761 4587
14630685 40263
8557367 44383
-5169082 1818
766487 8991
