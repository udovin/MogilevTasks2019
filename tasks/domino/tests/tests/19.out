1 2 2 1 1 81 1 1 1 2 3 1 1 1 3 181 1 68 2 1 1 1 1 1 2 1 2 2 1 3 1 1 1 2 2 1 1 1 3 1 1 1 1 2 2 1 1 3 1 1 1 3 2 1 1 1 1 1 1 1 1 1 1 1 1 2 5 1 2 2 2 1 1 1 1 1 1 1 1 1 2 2 1 1 2 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 1 2 1 1 1 1 1 1 237 1 1 1 1 1 3 1 1 1 363 1 1 1 4 1 1 1 2 1 1 1 1 4 1 1 1 2 1 1 2 1 1 1 3 1 2 282 1 1 2 1 2 1 1 2 1 1 1 2 2 1 1 213 2 1 2 1 1 1 1 1 1 2 1 1 1 1 1 1 1 1 308 1 1 2 2 1 1 1 1 3 1 2 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 1 3 1 1 1 1 1 1 1 1 1 1 1 3 2 1 1 1 1 1 1 1 1 1 2 1 1 2 1 1 1 2 1 1 1 1 2 1 1 1 1 2 1 1 1 1 3 5 1 178 1 1 1 1 1 1 1 2 1 1 1 1 6 1 2 2 1 1 2 1 1 2 1 1 1 1 1 1 1 1 220 1 2 2 1 1 1 1 1 2 1 1 3 1 1 1 2 1 2 2 1 1 1 2 1 1 2 1 1 1 1 1 2 1 2 1 2 1 3 3 1 1 138 1 1 2 2 1 1 1 1 1 2 1 1 1 1 1 1 1 1 1 221 1 1 1 1 164 1 2 1 1 1 1 1 1 303 2 1 2 2 1 1 1 1 1 2 2 2 1 1 1 1 1 1 1 1 2 1 1 1 3 1 1 1 1 1 2 1 1 1 133 1 1 5 2 1 1 1 1 1 1 3 23 2 1 1 2 1 1 4 1 1 2 148 1 2 2 1 326 1 1 1 1 1 2 1 1 1 1 1 1 2 3 1 1 1 1 2 2 4 1 4 99 1 1 3 1 1 1 2 1 313 1 1 1 1 1 4 1 1 
